<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Meena-Laptop" failures="1" tests="4" name="SalesForceLoginTest" time="31.455" errors="0" timestamp="2024-10-18T11:04:21 CDT">
  <testcase classname="com.automation.tests.LoginTests" name="TC1SalesforceLogin_BlankPwd" time="1.602"/>
  <testcase classname="com.automation.tests.LoginTests" name="TC2SalesforceLoginValid" time="4.002"/>
  <testcase classname="com.automation.tests.LoginTests" name="TC4A_ForgotPassword" time="1.509">
    <failure type="java.lang.AssertionError" message="expected [Weâ€™ve sent you an email with a link to finish resetting your password.] but found [We’ve sent you an email with a link to finish resetting your password.]">
      <![CDATA[java.lang.AssertionError: expected [Weâ€™ve sent you an email with a link to finish resetting your password.] but found [We’ve sent you an email with a link to finish resetting your password.]
at com.automation.tests.LoginTests.TC4A_ForgotPassword(LoginTests.java:123)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- TC4A_ForgotPassword -->
  <testcase classname="com.automation.tests.LoginTests" name="TC4B_ValidateLoginErrorMsg" time="1.994"/>
</testsuite> <!-- SalesForceLoginTest -->
